src/fonts.c

```diff
@@ -37,6 +37,42 @@ ALIGNED(4) const u8 gFontSmallNarrowLatinGlyphWidths[] = {
 };
 
 ALIGNED(4) const u16 gFontSmallLatinGlyphs[] = INCBIN_U16("graphics/fonts/small.latfont");
+#if POKEDEX_PLUS_HGSS == TRUE
```

Comment:
Since we're adding the Egg icon to the font itself, I would just modify the specific width.